lib/stoplight/light/runnable.rb

```diff
@@ -26,6 +27,20 @@ def run
         end
       end
 
+      def lock(color)
```

Comment:
Sorry, one final thought. Would you mind adding yard docs to the method? Since it's a public integrate, let's document what it returns, I suggest returning either self of color. I lean toward a color, but don't have a strong opinion about that

Reply:
I decided to return Light.